package.json

```diff
@@ -89,7 +89,8 @@
 		"prettier": "^2.7.1",
 		"pretty-quick": "^3.1.3",
 		"readdirp": "^3.6.0",
-		"turbo": "1.10.16"
+		"turbo": "latest",
+		"typescript": "^5.1.6"
```

Comment:
is this necessary? if so, could we use ^4 as our resolutions pin that (rrweb is broken with 5)